var/spack/repos/builtin/packages/palisade-development/package.py

```diff
@@ -29,10 +29,10 @@ class PalisadeDevelopment(CMakePackage):
     maintainers = ['wohlbier']
 
     version('feature-fixed-point-encoding',
-            branch='feature-fixed-point-encoding')
-    version('fppe-logreg',
-            branch='fppe-logreg')
-    version('master', branch='master', preferred=True)
+            branch='feature-fixed-point-encoding', submodules=True)
+    version('fppe-logreg-v1.0',
+            tag='fppe-logreg-v1.0', submodules=True)
```

Comment:
Is there any reason to version a feature branch instead of using just 1.0?

Reply:
Yes, this feature isn't available in a release yet. And 1.0 on the feature branch is a tag that is known to work while they continue development.